python/core/symbology-ng/qgsstyle.sip

```diff
@@ -110,6 +99,14 @@ class QgsStyle : QObject
      *  \return returns the success state of the operation
      */
     bool detagSymbol( StyleEntity type, QString symbol, QStringList tags );
+    //! clear the symbol from all attached tags
```

Comment:
Some oddness here - looks like there's a duplicate doxygen tag, and the proper one should use `/**` not `/*!`